public/endpoints.js

```diff
@@ -3,6 +3,9 @@ window.endpoints = {
   results: 'http://test_domain.com',
   graphql: 'http://test_domain.com',
   prefix: 'test_prefix.',
-  result_index: 'test_index.',
-  run_index: 'test_index.',
+  run_index: 'vn.run-data.',
+  run_toc_index: 'vn.run-toc.',
+  result_index: 'vn.result-data-sample.',
+  result_data_index: 'vn.result-data.',
```

Comment:
These five above don't seem like end points.  This seems like what we want the initial server load to give the dashboard.

Reply:
Yeah, "endpoints" might be better termed "server configuration". More obvious now that I split the two main indices. In my configuration API I put these in a "metadata" section separate from the "endpoints" (URI) section.

Reply:
Hey @dbutenhof ! I suppose this file is in `.gitignore`

Reply:
> Hey @dbutenhof, I suppose this file is in `.gitignore`

Yes, it is; although for some reason that pattern isn't working for me. Which in this case is fine as I had to add additional configuration keys.

Reply:
Actually, that's a good point; .gitignore is only for _untracked_ files. Maybe we should really have a `public/endpoints.example.js` that's tracked and updated, with the build looking for an untracked `public/endpoints.js` that's in .gitignore.

Reply:
Hmm; that, unfortunately, breaks the current end-to-end testing because the search test requires `endpoints.js` placeholders that match the internal test framework placeholders. (This is actually a problem if you run the tests with a "real" `endpoints.js`, too.) It's probably possible to get npm/yarn/umi to "build" the example file as the `endpoints.js` to use, but I don't know enough about the capabilities and structure of `package.json` to do that.

Reply:
I'll take a look into possibly using a mocked endpoint file for enabling e2e testing without the requirement of `endpoints.js`. @dbutenhof we should update the fields present in the README.md file at the root of the project directory.